src/test/resources/alma/HT017131400.json

```diff
@@ -0,0 +1,122 @@
+{
```

Comment:
This resource doesn't test 'inCollection'. It should be removed.

Reply:
The resource is removed, but it still exists in the tar.bz2. We keep it there nonetheless, bacause we will need it in the near.